### ISSUE TITLE
support aws shield

### ISSUE THREAD

**Issue by szuecs** [zalando-incubator/kube-ingress-aws-controller#202]
https://docs.aws.amazon.com/config/latest/developerguide/resource-config-reference.html#awsshield

**Comment by amila-ku** [comment]
i would like to work on this issue..

**Comment by szuecs** [comment]
Please share what you exactly want to build, if you figured it out. For example "user set per ingress annotation" or "operator starts the controller with a new flag `--aws-shield-id=myshield`".
Let us know if you need any help. We can guide you in the code, if needed.

**Comment by amila-ku** [comment]
I looked into the shield implementation guides and it only seems to require additional configuration when used as the shield advanced which is $3000 per month. Initially, I was under the impression shield can also be used in 'pay as you go' service. So now I don't think shield advanced is something I want to use.

**Comment by szuecs** [comment]
@amila-ku, fair enough, I never looked into it, but I agree the price model seems to be quite expensive and not what you should want to have. It seems also quite expensive to test it.
